.gitignore

```diff
@@ -12,3 +11,4 @@ docs/
 npm-debug.log
 /.idea/
 .env
+dist/*
```

Comment:
Why does `dist` get added to `gitignore`? This seems to break the current import API.